catroid/src/main/java/org/catrobat/catroid/sensing/CollisionDetection.java

```diff
@@ -44,19 +43,17 @@
 	private CollisionDetection() {
 	}
 
-	public static double checkCollisionBetweenLooks(Look firstLook, Look secondLook) {
+	public static boolean checkCollisionBetweenLooks(Look firstLook, Look secondLook) {
```

Comment:
Nice! 
Can you please also rename the function? areLookColliding(..) or smth similar. 
(In my opinion, look.collidesWith(look) and then the polygon check, would be the best)
But iff you want it to keep it that way it's also ok 👍

Reply:
Hmm. Should I then rename all other functions in this class as well? Maybe Kotlin will help here with some extension functions

Reply:
It's just more work, so you don't have to. 
I think extension functions are not necessary because we have access to class "Look" :smile: